src/components/ShareStructure/ShareStructures.vue

```diff
@@ -49,23 +51,27 @@ import {
 export default class ShareStructures extends Mixins(CommonMixin) {
   /** Edit Mode */
   @Prop({ default: true })
-  private isEditMode!: string
+  readonly isEditMode!: string
 
-  // Global Getters
+  // Global getters
   @Getter getNewResolutionDates!: string []
   @Getter getOriginalIA!: IncorporationFilingIF
   @Getter getShareClasses!: ShareClassIF[]
-  @Getter getOriginalSnapshot!: BusinessSnapshotIF[]
+  @Getter getOriginalSnapshot!: BusinessSnapshotIF
 
-  // Global Actions
+  // Global actions
   @Action setShareClasses!: ActionBindingIF
   @Action setShareStructureChanged!: ActionBindingIF
   @Action setEditingShareStructure!: ActionBindingIF
 
-  // Local Properties
-  private toggleResolutionDateDialog = false
+  // Local property
+  toggleResolutionDateDialog = false
```

Comment:
Why no `private` in this instance, is it being used/accessed outside the class?

Reply:
No 😞 It's because VS Code shows this property as unused when it's private. It's actually used only in the template.

I'm guessing Vue/TS are still working through this. IMHO it should be private (as the template should be able to "see" what's in the script, as it's part of the class/component, right?). But, trying to make VS Code happy here. I could go either way.

Reply:
For sure, i only ask out of curiosity! I've no problem with it.